.github/workflows/build_and_test.yml

```diff
@@ -40,23 +40,23 @@ jobs:
     secrets:
       gcloud-service-key: ${{ secrets.GCLOUD_SERVICE_KEY }}
 
-  build-torch-with-cuda:
-    name: "Build PyTorch with CUDA"
-    uses: ./.github/workflows/_build_torch_with_cuda.yml
-    needs: get-torch-commit
-    with:
-      # note that to build a torch wheel with CUDA enabled, we do not need a GPU runner.
-      dev-image: us-central1-docker.pkg.dev/tpu-pytorch-releases/docker/development:3.10_cuda_12.1
-      torch-commit: ${{needs.get-torch-commit.outputs.torch_commit}}
-      runner: linux.24xlarge
+  # build-torch-with-cuda:
```

Comment:
Can we add a bug or PR comment explaining why these are commented out? (Ditto for other commented sections)

Reply:
yea let me update